apollo-runtime/src/main/java/com/apollographql/android/cache/normalized/InMemoryCacheStore.java

```diff
@@ -0,0 +1,24 @@
+package com.apollographql.android.cache.normalized;
+
+import java.util.Collection;
+
+public final class InMemoryCacheStore extends CacheStore {
+
+  private final RecordSet recordSet;
```

Comment:
Now that I think about it, let's not include this at all in the `runtime`. The only thing using it is the tests, so I will move it there. We can cross the bridge of what default to ship with later.